### ISSUE TITLE
Deprecate this extension in favor of "Termination"

### ISSUE THREAD

**Issue by shakram02** [jeremyramin/terminal-plus#451]
Hello, please deprecate this extension since it seems abandoned, it'll be in the favor of the community.
Please also point to **[Termination](https://github.com/Fred-Barclay/Termination)** a fork of this project by @Fred-Barclay as it's active and in development ( and works! ).

A lot of people download your plugin as it is more famous and has high download numbers.

Thanks,

**Comment by pablougas** [comment]
Termination seems to be the same thing... except it actually works. Thanks @shakram02

**Comment by shakram02** [comment]
yup it's a fork of this project ( I updated the issue )

**Comment by elebumm** [comment]
Another alternative is this:

https://github.com/platformio/platformio-atom-ide-terminal

Does the exact same thing but works. I agree on completely deprecating this package.

**Comment by Fred-Barclay** [comment]
@elebumm Yep, platformio-atom-ide-terminal is actually our upstream and it works great! You could think of Termination as platformio-terminal styled to look like terminal-plus + a few extra features. :smile: 

(and thanks to @shakram02 for spreading the word!)
